scripts/README.md

```diff
@@ -30,7 +30,13 @@ python scripts/generate_train_scripts.py --experiment_group "drop-only"
 
 to generate bash scripts in `ROOT/train_scripts/`. Note additional optional arguments include `--dataset_config_path` and `--train_config_path` (see below).
```

Comment:
Let's be clear about slurm usage throughout, "generate *slurm* scripts"

Reply:
should state prereqs for using slurm script (access to slurm, conda env creation - though not necessarily specific steps required on *our* HPC to create it!)

Reply:
ah ok we should make it clear that this setups up slurm scripts for *our* HPC as there are some Baskerville specifics

Reply:
Have added these bits to the README